bundles/org.eclipse.passage.lic.licenses.model/model/licenses..genmodel

```diff
@@ -11,7 +11,7 @@
     importerID="org.eclipse.emf.importer.ecore" complianceLevel="8.0" copyrightFields="false"
     editPluginID="org.eclipse.passage.lic.licenses.edit" editorPluginID="org.eclipse.passage.lic.licenses.editor"
     language="" operationReflection="true" runtimePlatform="RCP" importOrganizing="true"
-    cleanup="true" oSGiCompatible="true">
+    pluginKey="licenses" cleanup="true" oSGiCompatible="true">
```

Comment:
Is this needed?

Reply:
Suppose so. But definitely in another MR - the one dedicated to `licenses..core` -> `licenses.core` renaming